### PR TITLE
modify rootfstype for nfs-based statelite with SN testcase

### PR DESCRIPTION
**Pull request by bybai**
This pull request is for issue #2920  :
1,  modify rootfstype=nfs
2,  print osimage definition
3,  get `showmount -e` output from SN

### CODE REVIEW

File: xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs [hunk 1]
```diff
@@ -42,6 +42,8 @@ check:rc==0
 cmd:if cat /etc/*release |grep SUSE >/dev/null;then xdsh $$SN 'service nfsserver restart'; elif cat /etc/*release |grep "Red Hat" >/dev/null;then xdsh $$SN 'service nfs restart'; fi
 check:rc==0
 
+cmd:xdsh $$SN 'showmount -e'
+
```
Comment (daniceexi) [inline xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs]: We don't check the result of xdsh, why did we add this cmd run? Display message?

File: xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs [hunk 2]
```diff
@@ -58,6 +60,12 @@ check:rc==0
 cmd:rootimgdir=`lsdef -t osimage __GETNODEATTR($$CN,os)__-__GETNODEATTR($$CN,arch)__-statelite-compute|grep rootimgdir|awk -F'=' '{print $2}'`; if [ -d $rootimgdir ]; then mv $rootimgdir $rootimgdir.regbak;fi
 check:rc==0
 
+cmd:lsdef -t osimage -o __GETNODEATTR($$CN,os)__-__GETNODEATTR($$CN,arch)__-statelite-compute
+check:rc==0
+
+cmd:chdef -t osimage -o __GETNODEATTR($$CN,os)__-__GETNODEATTR($$CN,arch)__-statelite-compute rootfstype=nfs
```
Comment (daniceexi) [inline xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs]: I'd like to ask whether the case could success without this change **rootfstype=nfs**?

File: xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs (hunk 1 above)
Comment (bybai) [inline xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs]: I would like to use this display message and debug.

File: xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs (hunk 2 above)
Comment (bybai) [inline xCAT-test/autotest/testcase/installation/reg_linux_statelite_installation_hierarchy_by_nfs]: This testcase is affected by its previous testcase reg_linux_statelite_installation_hierarchy_by_ramdisk.

**Pull request merged**